scripts/rotate_cloud_secrets.sh

```diff
@@ -0,0 +1,46 @@
+if brew ls --versions jq gh > /dev/null; then
+  continue
+else
```

Comment:
Would suggest making this line more portable, since future maintainers may not have brew on their OS.

```suggestion
if [ ! $(command -v gh) ] || [ ! $(command -v jq) ]; then
```